README.md

```diff
@@ -1,92 +1,38 @@
+<img src="https://img.shields.io/badge/-CSESoc-blue" />
+
 # Circles
+> see the [confluence workspace and links](https://compclub.atlassian.net/wiki/spaces/C/pages/1664385928/About+Circles) for further development documentation and a linktree to our jira boards, and more!
```

Comment:
A lot of these will be super nit picky grammar things, probs a capital here on 'see'

Reply:
Maybe 'for further development documentation, a linktree to our jira boards and more!' instead